### PR TITLE
fix #70726 erolub.com

### LINKED ISSUE #70726: erolub.com
### Issue URL (Ads)
[NSFW] [http://erolub.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Ferolub.com%2F)

### [NSFW] Screenshots
<details>
  <summary>Screenshot 1</summary>

  [Screenshot 1](https://reports-img.adguard.com/PA05kpq.jpg)
</details>

### System configuration

Information | value
--- | ---
Platform: | uBlock Origin
Browser: | Firefox
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/70726
Not sure if `||bcvideod.com^$third-party` can also be added, tho blocked initiator.

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -35,6 +35,8 @@ netfullfilmizle.com,filmyani.com,sonsaat.com.tr,okudunmuhaberi.com,spooor.com,ma
...
 !
+||erolub.com/templates/flay-elub/images/share.png
+erolub.com##.share-big
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:39]: I don't see the request `erolub.com/templates/flay-elub/images/share.png`.
`erolub.com##.share-big` why did you add it? Those share buttons are blocked by generic rules from AdGuard Social filter.

**Comment by zloyden** [comment]
> Not sure if `||bcvideod.com^$third-party` can also be added, tho blocked initiator.

You have to add this rule - it's the ad network called `BigClick`.

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:39]: Strange, I'll test again.

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt]: Hmm...confirmed. Something went wrong on my end. Reverted.

**Review by zloyden — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply SocialFilter/sections/specific.txt]: Ok:)
